content/docs/guides/node-exporter.md

```diff
@@ -0,0 +1,100 @@
+INFO[0000] Build context (go=go1.9.6, user=root@a67a9bc13a69, date=20180515-15:53:28)  source="node_exporter.go:83"
+INFO[0000] Enabled collectors:                           source="node_exporter.go:90"
+INFO[0000]  - boottime                                   source="node_exporter.go:97"
+...
+INFO[0000] Listening on :9100                            source="node_exporter.go:111"
+```
+
+## Node exporter metrics
+
+Once the node exporter is installed and running, you can verify that metrics are being exported by cURLing the `/metrics` endpoint:
+
+```bash
+curl http://localhost:9100/metrics
+```
+
+You should see output like this:
+
+```
+# HELP go_gc_duration_seconds A summary of the GC invocation durations.
+# TYPE go_gc_duration_seconds summary
+go_gc_duration_seconds{quantile="0"} 3.8996e-05
+go_gc_duration_seconds{quantile="0.25"} 4.5926e-05
+go_gc_duration_seconds{quantile="0.5"} 5.846e-05
+# etc.
+```
+
+Success! The node exporter is now exposing metrics that Prometheus can scrape, including a wide variety of system metrics further down in the output (prefixed with `node_`). To view those metrics (along with help and type information):
+
+```bash
+curl http://localhost:9100/metrics | grep "node_*"
+```
+
+## Configuring your Prometheus instances
+
+Your locally running Prometheus instance needs to be properly configured in order to access node exporter metrics. The following [`scrape_config`](../prometheus/latest/configuration/configuration/#<scrape_config>) block will tell Prometheus that scrape from the node exporter via `localhost:9100`:
+
+```yaml
+scrape_configs:
+- job_name: 'node'
+  static_configs:
+  - targets: ['localhost:9100']
+```
+
+To install Prometheus, [download the latest release](/download) for your platform,
+
+```bash
+tar xvf prometheus-*.*-amd64.tar.gz
+cd prometheus-*.*
+```
+
+Once Prometheus is installed you can start it up, using the `--config.file` flag to point to the Prometheus configuration that you created:
+
+```bash
+./prometheus --config.file=./prometheus.yml
+```
+
+## Exploring node exporter metrics through the Prometheus expression browser
+
+Now that Prometheus is scraping metrics from a running node exporter instance, you can explore those metrics using the Prometheus UI (aka the [expression browser](/docs/visualization/expression-browser)). Navigate to `localhost:9090/graph` in your browser and use the main expression bar at the top of the page to enter expressions, which looks like this:
+
+![](/assets/prometheus-expression-bar.png)
+
+Metrics specific to the node exporter are prefixed with `node_` and include metrics like `node_cpu_seconds_total` and `node_exporter_build_info`.
+
+Click on the links below to see some example metrics:
+
+* [`node_cpu_seconds_total{mode="system"}`](http://localhost:9090/graph?g0.range_input=1h&g0.expr=node_cpu_seconds_total%7Bmode%3D%22system%22%7D&g0.tab=1)
```

Comment:
Anything with a _total needs a rate(x[1m]) around it to be useful

A few words about what these metrics mean would be useful